.github/workflows/deploy-staging.yml

```diff
@@ -61,3 +64,33 @@ jobs:
           version: v1.21.0 # specify kubectl binary version explicitly
           command: rollout status deployment/annotation-ui-staging
 
+  ogbv-ml-rest:
+    runs-on: ubuntu-latest
+    needs: changes
+    if: ${{ needs.changes.outputs.ogbv-ml-rest == 'true' }}
+    steps:
+      -
+        name: Checkout 
+        uses: actions/checkout@v2
+        with: 
+          ref: development
+      - 
+        name: Declare some variables
+        id: vars
+        shell: bash
+        run: |
+          echo "setting variables"
+          echo "::set-output name=sha_short::$(git rev-parse --short HEAD)"
+          
+      -
```

Comment:
@Bhargav-Dave looks like this indentation/formatting issue is there on all steps of your job. can you fix and push?

Reply:
Sure, doing that